src/routes/sections/FormSection.svelte

```diff
@@ -1,29 +1,62 @@
+<script>
+  import emailjs from "@emailjs/browser";
+
+  function sendEmail(e) {
+    emailjs
+      .sendForm(
+        "contact_service",
+        "contact_form",
+        e.target,
+        "Pl4Sqn7Ye5JukP1tw"
```

Comment:
Ojo acá!
No poner nunca las api keys directo en el codigo por más que sean public keys..

Todas las api keys y otras cosas privadas van el archivo `.env`.
Este nunca se gitea, entonces nunca llega a github.

Aca un articulo de como manejar las variables de `.env` en SvelteKit: https://www.okupter.com/blog/environment-variables-in-sveltekit